Day_10

Garret-vin opened the pull request:
add sha256 hashing for passwords

Garret-vin committed:
Day_10

add sha256 hashing for passwords

Garret-vin committed:
Day_10

add sha256 hashing for passwords
replaced all statements to preperedStatements

Sofasmile commented on src/main/java/com/controller/AddUserServlet.java:
```diff
@@ -47,7 +48,8 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             req.getRequestDispatcher("/addUser.jsp").forward(req, resp);
         } else if (password.equals(confirmPassword)) {
-            User user = new User(login, email, password, role);
+            String sha256Password = DigestUtils.sha256Hex(password);
```
```suggestion
            String encryptedPassword = DigestUtils.sha256Hex(password);
```
in all classes

Garret-vin committed:
Day_10

add sha256 hashing for passwords
replaced all statements to preperedStatements
(up)
